rotkehlchen/tests/unit/decoders/votium.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+
+from rotkehlchen.accounting.structures import (
+    Balance,
+    HistoryBaseEntry,
+    HistoryEventSubType,
+    HistoryEventType,
+)
+from rotkehlchen.assets.asset import EthereumToken
+from rotkehlchen.chain.ethereum.decoding.decoder import EVMTransactionDecoder
+from rotkehlchen.chain.ethereum.structures import EthereumTxReceipt, EthereumTxReceiptLog
+from rotkehlchen.chain.ethereum.types import string_to_ethereum_address
+from rotkehlchen.constants.assets import A_ETH
+from rotkehlchen.db.ethtx import DBEthTx
+from rotkehlchen.fval import FVal
+from rotkehlchen.types import EthereumTransaction, Location, deserialize_evm_tx_hash
+from rotkehlchen.user_messages import MessagesAggregator
+from rotkehlchen.utils.hexbytes import hexstring_to_bytes
+
+
+@pytest.mark.parametrize('ethereum_accounts', [['0x362C51b56D3c8f79aecf367ff301d1aFd42EDCEA']])  # noqa: E501
+def test_votium_claim(database, ethereum_manager):
+    """Data for trade taken from
```

Comment:
This is not really a trade, right? I guess this is copy pasta error. But can ignore for now. Let's remember to change it in the future.